temporalio/exceptions.py

```diff
@@ -65,10 +68,14 @@ def __init__(
         message: str,
         *details: Any,
         type: Optional[str] = None,
-        non_retryable: bool = False
+        non_retryable: bool = False,
```

Comment:
Since we control the default value can we change this to `retryable: bool = True` to improve readability?

Reply:
I would like the init param names to match the property names. Currently we expose Java [isNonRetryable](https://www.javadoc.io/static/io.temporal/temporal-sdk/1.14.0/io/temporal/failure/ApplicationFailure.html#isNonRetryable()), Go [NonRetryable](https://pkg.go.dev/go.temporal.io/sdk/internal#ApplicationError.NonRetryable), TS [nonRetryable](https://typescript.temporal.io/api/classes/common.ApplicationFailure#nonretryable), etc.